src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -59,7 +68,14 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
     val serialized = messages.map(aw => Try {
```

Comment:
@krasserm this `Try` will catch serialization exceptions, which will end up as `WriteMessageRejected`. That's a nice thought, but it will result in holes in the sequence numbers. Having sequence numbers without holes is very convenient for detecting delayed events (avoid wrong order).

Is it alright for you that we don't support rejections, i.e. we fail the whole write for serialization issues?